lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
-# Lesson Name
-
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/outline/) for guidance on writing lesson outline.
+# Testing in Jest
 
 ## Lesson Information
 
 ### Resource(s)
 
-_Insert any resources you used while researching to help plan your lesson._
+- [jest official documentation](https://jestjs.io/)
+
 
 ### Learning Standards
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Resources/outcomes-standards-objectives/#learning-standards) for guidance on writing learing standards.
 
-1. ...
-2. ...
-3. ...
+1. What is Jest
+2. Installing Jest
+3. Running Jest Test
+4. The `test()` function
+5. The `expect()` function and matchers
+6. False Positives in Jest with Asynchronous Code
+7. Testing Async Code - Async Callbacks
+8. Testing Async Code - Promises
+9. Mocking functions with `jest.fn()`
+10. Mocking modules with `jest.mock()`
+
 
 <hr>
 
-## Exercise # 1: _Insert exercise title here_
+## Exercise # 1: What is Jest
 
 ### Which course outcomes will be covered by this exercise?
 
-1. Learners will be able to...
+1. Learners will obtain a general understanding of testing and why it is important. 
```

Comment:
Akil, try to think about outcomes as things that learners will be able to (tangibly) do. E.g. It's hard to know how much someone "understands" something if you ask them, "Hey, do you understand algebra?" — you might get a yes, or no, or anything in between. 

One way we try to define concrete outcomes is to use [Bloom's Taxonomy](https://tips.uark.edu/blooms-taxonomy-verb-chart/) verbs to speak specifically as to what learners should be able to do. In this case this outcome could be something like: "Learners will be able to describe how testing in code works and identify why testing is needed in an application."  

Notice I use "describe" and "identify" both at the same basic Bloom level but both showcasing me that learner's understanding will be at a basic level. We're expecting them to say what testing is and point out the need.